scripts/shared/resources/nftables-cm.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
If this is for packet filter configuration, let’s name the file accordingly; it’s used for nftables currently but doesn’t seem to be limited to that.

Reply:
Should we have a generic submariner customization configmap instead of a specific cm for packetfilter?  maybe other submariner components can extend it if needed